code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -181,14 +180,13 @@
 	force = 10
 	slot_flags = 0
 	max_shells = 50
-	caliber = "5.56mm"
+	caliber = "5.45mm"
 	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 1, TECH_ILLEGAL = 2)
 	slot_flags = SLOT_BACK
-	ammo_type = "/obj/item/ammo_casing/a556" // Is this really needed anymore?

```

Comment:
As mentioned above, not sure what precisely this does, but I'm not sure why it's being removed.